sycl/source/detail/graph/graph_impl.hpp

```diff
@@ -561,10 +559,12 @@ class graph_impl : public std::enable_shared_from_this<graph_impl> {
   /// Device associated with this graph. All graph nodes will execute on this
   /// device.
   sycl::device MDevice;
+
+  using RecQueuesStorage =
```

Comment:
nit: I would name it `RecordingQueueTy` or `RecordingQueueType`. We have no common coding style, but we already used the aforementioned naming schema in other places.